modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -417,8 +415,27 @@ class ONNXtest : public ::testing::Test {
         }
     }
 
-    void useModel(const std::string& model_name) {
+    // The function sets path to a model and a path to input test data,
```

Comment:
The function sets the path to the mode and test data

Reply:
Fixed.

Reply:
Removed.